### PR TITLE
More useful methods are added to the ControlBase class.

### PR DESCRIPTION
**Pull request by aannenko**
I was using these methods' analogues while automating other projects using Selenium so I'm pretty sure they can come in handy not only to me.

### CODE REVIEW

**Comment by icnocop** [comment]
Thank you!

**Comment by aannenko** [comment]
@icnocop @FantasticFiasco Hi guys! Can you help me a bit? Do you have any idea about why my cuit test are no longer recognized as tests if I switch the original CUITe.dll that is added to the project by NuGet to my own that contains a new function which I have to test?


I just change the dll to my own and then rebuild the project. Rebuild works just fine for the project and the whole solution after I put my own dll.

In other words, is there a proper way to build CUITe so that it could be treated like the one from NuGet?

**Comment by FantasticFiasco** [comment]
I think the easiest way is to run _Build.bat_ in the root directory since it will produce a couple of NuGet packages. Add the directory with the NuGet packages as a local NuGet source in Visual Studio and install the packages as you would with any other package. I think that is a better solution than having to replace specific assemblies.

**Comment by aannenko** [comment]
Still no luck... I tried to attach all the CUITe.dll files that are created by the build.bat to my project and each of them makes my test not recognized. Ok, forget it, I'll just create a new project in the CUITe solution and put my test in it.
